transformations/dyslexia_words_swap/transformation.py

```diff
@@ -0,0 +1,42 @@
+import os
+import json
+import spacy
+from interfaces.SentenceOperation import SentenceOperation
+from tasks.TaskTypes import TaskType
+
+
+class DyslexiaWordsSwap(SentenceOperation):
+
+    tasks = [TaskType.TEXT_CLASSIFICATION, TaskType.TEXT_TO_TEXT_GENERATION]
+    languages = ["en"]
+    keywords = ["lexical", "rule-based", "external-knowledge-based", "aural", "possible-meaning-alteration", "low-precision", "low-coverage", "low-generations"]
+
+    def __init__(self, seed=0, max_outputs=1):
+        super().__init__(seed=seed, max_outputs=max_outputs)
+
+        self.nlp = spacy.load("en_core_web_sm")
```

Comment:
Please use global spacy instead of loading a new one. 

self.nlp = spacy_nlp if spacy_nlp else spacy.load("en_core_web_sm")

Refer to other transformations for more detail.

Reply:
Fixed ! Thanks !